.github/workflows/ecs.yml

```
@@ -0,0 +1,50 @@
+name: Deploy to ECS
+
+on: 
+  workflow_call:
+    inputs:
+      iam_role:
+        required: true
+        type: string
+      region:
+        required: false
+        type: string
+        default: us-east-1
+      ecs-cluster:
+        required: true
+        type: string
+      files:
+        required: false
+        type: string
+        default: docker-compose.yml:docker-compose.ecs.yml
+      env:
+        required: false
+        type: string
+        default: ''
+env:
+  GIT_SHA: ${{ github.sha }}
+
+permissions:
+      id-token: write
+      contents: read
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - run: |
+        echo ${{ inputs.env }} >> $GITHUB_ENV
```

This can be single-line

```suggestion
    - run: echo ${{ inputs.env }} >> $GITHUB_ENV
```